packages/angular/.editorconfig

```diff
@@ -0,0 +1,9 @@
+# http://editorconfig.org
+
+[*]
+charset = utf-8
+end_of_file = lf
+indent_style = tab
```

Comment:
is this purposely different from the root editor config?

Reply:
It was ... Given that I've formatted everything with Prettier now I'm guessing they'll be in sync, and this is unnecessary